script/test/fixtures/multiple-compose-files/output-k8s.json

```diff
@@ -89,31 +79,28 @@
                 "ports": [
                   {
                     "containerPort": 9001
-                  },
-                  {
-                    "containerPort": 9001
                   }
                 ],
                 "env": [
                   {
-                    "name": "DB_DBID",
-                    "value": "openshift"
+                    "name": "DB_HOST",
+                    "value": "kubernetes"
```

Comment:
The value should remain the same right? I mean the value should still be `openshift` right?